src/client/src/orangehrmRecruitmentPlugin/components/CandidateActionLayout.vue

```diff
@@ -0,0 +1,152 @@
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-card-container">
+    <oxd-text tag="h6" class="orangehrm-main-title">
+      {{ title }}
+    </oxd-text>
+    <oxd-divider />
+    <oxd-form :loading="loading" v-bind="$attrs">
+      <oxd-form-row>
+        <oxd-grid :cols="3">
+          <oxd-grid-item>
+            <oxd-input-field
+              v-model="candidate.candidateName"
+              :label="$t('recruitment.candidate')"
+              readonly
+            />
+          </oxd-grid-item>
+          <oxd-grid-item class="--offset-row-2">
+            <oxd-input-field
+              v-model="candidate.vacancyName"
+              :label="$t('recruitment.vacancy')"
+              readonly
+            />
+          </oxd-grid-item>
+          <oxd-grid-item class="--offset-row-2">
+            <oxd-input-field
+              v-model="candidate.hiringManagerName"
+              :label="$t('recruitment.hiring_manager')"
+              readonly
+            />
+          </oxd-grid-item>
+          <oxd-grid-item class="--offset-row-2">
+            <oxd-input-field
+              v-model="recruitmentStatus"
+              :label="$t('recruitment.current_status')"
+              readonly
+            />
+          </oxd-grid-item>
+        </oxd-grid>
+      </oxd-form-row>
+
+      <slot></slot>
+    </oxd-form>
+  </div>
+</template>
+
+<script>
+import {APIService} from '@/core/util/services/api.service';
+
+const candidateModel = {
+  candidateName: '',
+  vacancyName: '',
+  hiringManagerName: '',
+  status: null,
+};
+
+export default {
+  name: 'CandidateActionLayout',
+  inheritAttrs: false,
+  props: {
+    candidateId: {
+      type: Number,
+      required: true,
+    },
+    title: {
+      type: String,
+      required: true,
+    },
+    loading: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  emits: ['update:loading'],
+  setup(props) {
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      `/api/v2/recruitment/candidates/${props.candidateId}`,
+    );
+
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      candidate: {...candidateModel},
+      statuses: [
+        {id: 1, label: this.$t('recruitment.application_initiated')},
+        {id: 2, label: this.$t('recruitment.shortlisted')},
+        {id: 3, label: this.$t('recruitment.rejected')},
+        {id: 4, label: this.$t('recruitment.interview_scheduled')},
+        {id: 5, label: this.$t('recruitment.interview_passed')},
+        {id: 6, label: this.$t('recruitment.interview_failed')},
+        {id: 7, label: this.$t('recruitment.job_offered')},
+        {id: 8, label: this.$t('recruitment.offer_declined')},
+        {id: 9, label: this.$t('recruitment.hired')},
```

Comment:
@SharukaNP any idea

Reply:
If these are for the drop down for actions we don't need since now the API data would be translated.

Reply:
https://github.com/orangehrm/orangehrm/pull/1350 can get this to this PR